modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BatchEngineBundleTrackerTest.java

```diff
@@ -69,6 +76,18 @@ public class BatchEngineBundleTrackerTest {
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
+	@BeforeClass
+	public static void setUpClass() {
+		_originalStopWatch = ReflectionTestUtil.getAndSetFieldValue(
```

Comment:
@ccorreagg maybe we can apply this to other tests that followed the same pattern?

Reply:
Yes, I will create a new PR for it